src/bootstrap/config.rs

```diff
@@ -372,14 +368,15 @@ impl Target {
         target
     }
 }
+
 /// Structure of the `config.toml` file that configuration is read from.
 ///
 /// This structure uses `Decodable` to automatically decode a TOML configuration
 /// file into this format, and then this is traversed and written into the above
 /// `Config` structure.
 #[derive(Deserialize, Default)]
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
-struct TomlConfig {
+pub struct TomlConfig {
```

Comment:
Please make this `pub(crate)` so it can only be used from `minimal_config`.

Reply:
This was for the `src/bootstrap/config/tests.rs` refactoring. I will refactor it again and keep this struct private.